### DISCUSSION TITLE
TFT35 E3 V3.0  Hotend temp display limit change

### DISCUSSION THREAD

**Discussion by SamppaD78** [bigtreetech/BIGTREETECH-TouchScreenFirmware#2498] [Q&A]
I upgraded my printer to be able to handle higher temperatures, but my TFT35 screen is still limiting hotend max temp to 275C. I have compile Marlin firmware and it works in Marlin mode,so In Marlin mode I can adjust temperature beyond 275C. In TFT touch screen mode this doesn't work,due to configuration issue. As i found out,there is multiple places in TFT firmware configuration files than needs changes. Does anyone know where can i locate  TFT files that needs to be modified? All I can find is config.ini, in Copy to SD Card root directory to update file, and in TFT-> src->config.ini..made changes but after uploading still temperature limit is set to 275C.
If anyone has the time to post a guide or  file names that need to be changed and help me change this temp limit it will be highly appreciated.

**Comment by oldman4U** [comment]
Check closed tickets.

**Comment by SamppaD78** [comment]
Hi. I have done that 1309 closed tickets and haven't found ticket that is related to changing hotend max temperature on a TFT35 or a similar TFT screen. Can you provide any link to the actual issue? Thanks

**Comment by SamppaD78** [comment]
I found the way to set temperature.
